packages/superset-ui-encodable/src/fillers/inferFieldType.ts

```diff
@@ -0,0 +1,13 @@
+import { ChannelType } from '../types/Channel';
+import { isXOrY } from '../typeGuards/Channel';
+import { Type } from '../types/VegaLite';
+
+const temporalFieldNames = new Set(['time', 'date', 'datetime', 'timestamp']);
```

Comment:
is this a robust way to infer temporal? useful proxy and not sure of anything better 🤔

Reply:
I agree it is not robust and very cheap heuristics. Just offer a bit smarter inference than defaulting to `quantitative` all the time.